core/types/celo_additions.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+func (h *Header) IsIstanbulHeader() bool {
```

Comment:
I'd suggest instead just returning the deserialized extra data, so something like. Then we can use that function wherever we were previously calling `ExtractIstanbulExtra`
```
func (h *Header) IstanbulExtra() (*IstanbulExtra, error) {
```

Reply:
Done in [377d8bf](https://github.com/celo-org/celo-blockchain/pull/1961/commits/377d8bfa57c5d7a9b6753b1c8629cc753a218c52)